retrofit/src/main/java/retrofit2/Retrofit.java

```diff
@@ -542,6 +542,11 @@ public Builder callbackExecutor(Executor executor) {
       return this;
     }
 
+    /** Returns a modifiable list of call adapter factories. */
+    public List<CallAdapter.Factory> adapterFactories() {
```

Comment:
This should be called `callAdapterFactories()`

Reply:
Renamed in 57a39e68.